setup.py

```diff
@@ -50,10 +51,20 @@
 
 # we use pip requirements files to improve Docker layer caching
 
+# git+git@github.com:commonlims/django-templatetag-sugar.git@master#egg=django_templatetag_sugar
+GIT_REGEX = re.compile(r'^git+.*/(.*)\.git.*')
+
+
+def parse_req(line):
+    m = GIT_REGEX.search(line)
+    if m:
+        return m.group(1)
+    return line
+
 
 def get_requirements(env):
     with open(u'requirements-{}.txt'.format(env)) as fp:
-        return [x.strip() for x in fp.read().split('\n') if not x.startswith('#')]
+        return [parse_req(x.strip()) for x in fp.read().split('\n') if not x.startswith('#')]
```

Comment:
Is this handling reqs of both types?  1) django-debug-toolbar>=1.9.1,<1.10.0, and 2)  git+git@github.com:commonlims/django-templatetag-sugar.git@master#egg=django_templatetag_sugar?

Reply:
Yes it will - kind of. This is an ugly temporary hack to support git requirements without changing too much. So the setup.py file will just know that it depends on `django-templatetag-sugar`, while it will be installed in the makefile with information on from where to get it (since it's not pypi).